orchestra/admin.py

```diff
@@ -25,6 +25,7 @@
 from orchestra.models import TaskAssignment
 from orchestra.models import TimeEntry
 from orchestra.models import Todo
```

Comment:
high-level: i had imagined the QA interface would be some admin interface or some other screen in Orchestra. I don't quite get the flow---who loads which screen to do QA, and where does it end up showing up?

Reply:
It shows up in the customer success managers launch boost task.

Reply:
It shows up on the task page only if you are a project admin.